.github/workflows/ci_weekly.yml

```diff
@@ -10,6 +10,5 @@ on:
 jobs:
   donothing:
     runs-on: ubuntu-latest
-    if: false
     steps:
       - run: echo "Skipped"
```

Comment:
No-op workflow for testing, the `if: false` triggered errors and it looks harmless to have a workflow_dispatch job run the "echo" step